src/test/java/org/jenkinsci/plugins/workflow/cps/global/WorkflowLibRepositoryTest.java

```diff
@@ -124,6 +124,9 @@ public void userDefinedGlobalVariable() throws Exception {
                         "  echo 'title was '+config.title",
                         "}")
                         , "\n"));
+                FileUtils.writeStringToFile(new File(vars, "acmeClass.groovy"), StringUtils.join(Arrays.asList(
+                        "@groovy.transform.Field int answer = 42")
+                        , "\n"));
```

Comment:
At @kohsuke’s suggestion, verifying some other previously untested idioms which we were concerned might get accidentally broken.